index.d.ts

```diff
@@ -78,17 +78,26 @@ export interface ClientOpts {
 	pingInterval?: number,
 	preserveBuffers?: boolean,
 	reconnect?: boolean,
+	// jitter for non-tls connections
+	reconnectJitter?: number,
+	// jitter for tls connections
+	reconnectJitterTLS?: number,
+	// handler for custom tls connection
```

Comment:
not sure about this comment: "// handler for custom tls connection"

Reply:
removed these comments as the README file contains the doc for this.